.metadata/.log

```diff
@@ -0,0 +1,28 @@
+!SESSION 2016-09-07 01:53:34.635 -----------------------------------------------
+eclipse.buildId=4.6.0.I20160606-1100
+java.version=1.8.0_65
+java.vendor=Oracle Corporation
```

Comment:
only relevant files should be committed. pls make sure gitignore is configured properly